### PR TITLE
Homework 5

### PR DESCRIPTION
**Pull request by ghost-lv**
part 1

### CODE REVIEW

**Review by natalia-kozak — APPROVED** [review]

File: src/com/lits/hw4/Array.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import java.util.Random;
 
-public class Array {
+class Array {
```
Comment (Volodymyr95) [inline src/com/lits/hw4/Array.java:5]: why default?

File: src/com/lits/hw5/Array.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package com.lits.hw5;
+
+import java.util.Random;
+
+class Array {
```
Comment (Volodymyr95) [inline src/com/lits/hw5/Array.java:5]: The Arrayrray is an abstraction
not correct

**Review by AndrewRorat — APPROVED** [review]

File: src/com/lits/hw4/Array.java (hunk 1 above)
Comment (ghost-lv) [reply src/com/lits/hw4/Array.java:5]: this file is form old h/w

File: src/com/lits/hw5/Array.java (hunk 2 above)
Comment (ghost-lv) [reply src/com/lits/hw5/Array.java:5]: don't understood

**Pull request merged**